projects/ngx-maplibre-gl/src/lib/map/map.component.ts

```diff
@@ -370,7 +369,7 @@ export class MapComponent implements OnChanges, OnDestroy, MapEvent {
       }
     });
 
-    afterRender(() => {
+    afterNextRender(() => {
```

Comment:
Are you sure about this change? I'm not sure if the failing e2e tests are related to this change or not, but it seems like a risky change...